src/schema/v2/lot.ts

```diff
@@ -0,0 +1,277 @@
+import { GraphQLObjectType, GraphQLString } from "graphql"
+import { pageable } from "relay-cursor-paging"
+import { ResolverContext } from "types/graphql"
+import { connectionWithCursorInfo } from "./fields/pagination"
+import { InternalIDFields } from "./object_identification"
+import SaleArtwork, { SaleArtworkType } from "./sale_artwork"
+
+const LotType = new GraphQLObjectType<any, ResolverContext>({
+  name: "Lot",
+  fields: () => ({
+    internalID: {
+      type: GraphQLString,
+      resolve: ({ saleArtwork }) => saleArtwork._id,
+    },
+    saleArtwork: {
+      type: SaleArtworkType,
+      resolve: ({ lot, saleArtwork }) => {
+        return saleArtwork
+      },
+    },
+    // internalID: ({saleArtwork}) => saleArtwork.id
+    // ...InternalIDFields,
+  }),
+})
+
+export const auctionLotConnection = connectionWithCursorInfo({
+  nodeType: LotType,
+})
+
+export const watchedLotsConnection = {
+  description: "A list of lots a user is watching",
+  type: auctionLotConnection.connectionType,
+  args: pageable(),
+  resolve: async (parent, args, context) => {
+    // rough approach:
+
+    // get Sale Artworks
+    // query gravity api/v2/sale_artworks?included_watched_artworks=true
+
+    // get lot state for each sale artwork from causality (ex query:
+    // query {
+    //   lots( ids: saleArtworks.map(sa => sa.id) ) {
+    //    # We probably have to select all graphql fields here?
+    //   }
+    // }
+
+    // zip them together and return with full necessary relay fields
+    context._lot = lotStub
+
+    return {
+      totalCount: 1,
+      pageCursors: {},
+      edges: [{ node: { saleArtwork: saleArtworkStub, lot: lotStub } }],
```

Comment:
here is our only field actually using the new Lot Type. its resolver will need to call gravity and then causality to assemble the data as pseudocoded above.